Initial proposal for visual redesign (#16)
osu-tournament-rating/otr-wiki: README.md

<table>
  <tr align="center">
    <td align="center" colspan=2 width="800">
      </br>
      <b>osu! Tournament Rating (o!TR)</b></br></br>
      Welcome to the official o!TR wiki!</br>
      o!TR is a fully-featured rating system & data visualization suite for osu! tournaments.</br></br>
      In this wiki you can find detailed documentations on all of the software that make up this platform, as well as important articles and guides that aim to help understand how to use the platform!</br></br>
      FAQ • Glossary</br>

>       Twitter • Discord • Github • Website

Author: Glossary was an idea to do something similar to the [osu!wiki sitemap](https://osu.ppy.sh/wiki/en/Sitemap). I thought it would be worth to include a straightforward list of links to all docs on the wiki, but I can see how it defeats the purpose of all the nice formatting. I'll leave this one up to you

>       Twitter • Discord • Github • Website

Author: Also I think it might be worth moving the FAQ out of `/algorithm` and giving it a dedicated spot. If we do that I'm not sure what would take its place in `/algorithm` though.

      </br></br>
    </td>
  </tr>
  <tr>
    <td align="center" width="400">
      </br>
      <b>Development</b></br></br>
      Looking to develop a third party program utilizing the platform, or considering contributing to the o!TR project?</br></br>
      <a href=api/README.md>o!TR API</a> • 
      <a>Ratings Processor</a> • 
      <a href=contribution/README.md>Contribution Guidelines</a> • 
      <a href=contribution/contributors/en.md>Contributors</a>
      </br></br>
    </td>
    <td align="center" width="400">
      </br><b>Website</b></br></br>

>       The o!TR website is the main access point of the platform (blah)(blah)(blah)(blah)(blah)(blah)

Author: Just so you know, traditional markdown formatting doesn't work inside of HTML elements. Therefore that embedded link would have to be written `<a href=https://otr.stagec.xyz/unauthorized>o!TR website</a>` and the last sentence would have to be wrapped in `<i></i>` tags for italics.

      </br></br>
    </td>
  </tr>
  <tr>
    <td align="center" width="400">
      </br>
      <b>Algorithm</b></br></br>
      The rating system utilizes quite a bit of complex math to perform calculations. For those curious, we break down the process and rationale in detail.</br></br>
      <a href=algorithm/README.md>General Theory</a> • 
      <a href=algorithm/ratings.md>Ratings</a> • 
      <a href=algorithm/tournaments.md>How We Handle Tournaments</a>
      </br></br>
    </td>
    <td align="center" width="400">
      </br><b>Help</b></br></br>

>       Find answers to frequently asked questions here! Have a question not answered here on the wiki, or want to suggest an improvement? Join the discord!</br></br>

Author: Linked github discussion pages for each repo, I think it makes the most sense?

<table>
  <tr align="center">
    <td align="center" colspan="2" width="800">
      </br>
      <b>osu! Tournament Rating (o!TR)</b>
      </br></br>
      Welcome to the official o!TR wiki!
      </br>
      o!TR is a fully-featured rating system & data visualization suite for osu! tournaments.
      </br></br>
      In this wiki you can find information about all of the tools that make up o!TR. We also host important articles, guides and resources that aim to help users understand how to use the platform.
      </br></br>

>       <a href="algorithm/tournaments.md">FAQ</a> • 

Reviewer: ```suggestion
      <a href="algorithm/README.md">FAQ</a> • 
```
Linking here would be more general for rating-related questions.

      <a href="https://discord.gg/R53AwX2tJA">Discord</a> • 
      <a href="https://github.com/osu-tournament-rating">Github</a> • 
      <a href="https://otr.stagec.xyz/unauthorized">Website</a>
      </br></br>
    </td>
  </tr>
  <tr>
    <td align="center" width="400">
      </br>
      <b>Development</b>
      </br></br>
      Looking to develop a third party program utilizing the platform, or considering contributing to the o!TR project?
      </br></br>
      <a href="api/README.md">o!TR API</a> • 
      <a>Ratings Processor</a> • 
      <a href="contribution/README.md">Contribution Guidelines</a> • 
      <a href="contribution/contributors/en.md">Contributors</a>
      </br></br>
    </td>
    <td align="center" width="400">
      </br>
      <b>Website</b>
      </br></br>
      The <a href="https://otr.stagec.xyz/unauthorized">o!TR website</a> is the official user-facing application in which you can seamlessly access your dashboard, player profiles, tournament information, and more! <i>Currently in a pre-release, whitelist only phase.</i>
      </br></br></br>
    </td>
  </tr>
  <tr>
    <td align="center" width="400">
      <b>Algorithm</b>
      </br></br>

>       The rating system utilizes quite a bit of complex math to perform calculations. For those curious, we break down the process and rationale in detail.

Reviewer: ```suggestion
      The rating system makes use of some complicated mathematics to analyze tournament data. For those curious, we break down the process and describe the rationale here.
```

      </br></br>
      <a href="algorithm/README.md">General Theory</a> • 
      <a href="algorithm/ratings.md">Ratings</a> • 

>       <a href="algorithm/tournaments.md">How We Handle Tournaments</a>

Reviewer: ```suggestion
      <a href="algorithm/tournaments.md">Collecting Tournament Data</a>
```
